src/biblenotify/core/notifications.py

```diff
@@ -96,7 +98,43 @@ def loadChapter(self, location: str) -> dict:
 
         contents_json = json.loads(contents_string)
 
+        text = self.highlightVerses(contents_json)
+
         return {
-            "text": contents_json["read"][0]["text"],
+            "text": text,
             "place": contents_json["read"][0]["chapter"]
         }
+
+    @Slot(str, result="QVariant")
+    def highlightVerses(self, contents_json):
+        # assuming every second verse is preceded by a special delimeter ;
+        # another delimeter should be used since this delimeter have too many edge cases
+        list = re.split(r";", verse["verse"])
```

Comment:
``list`` is a Python keyword, so this variable should be renamed.

Reply:
With removal of two verses, no need of this split i think. Reaffirmed not to use keyword for a variable again.